CMakeLists.txt

```diff
@@ -534,78 +539,26 @@ if(WITH_RADOSGW_BEAST_FRONTEND)
 	list(APPEND BOOST_COMPONENTS coroutine context)
 endif()
 
-if (WITH_SYSTEM_BOOST)
+set(Boost_USE_MULTITHREADED ON)
+# require minimally the bundled version
+if(WITH_SYSTEM_BOOST)
   if(ENABLE_SHARED)
     set(Boost_USE_STATIC_LIBS OFF)
   else()
     set(Boost_USE_STATIC_LIBS ON)
   endif()
+  find_package(Boost 1.61 COMPONENTS ${BOOST_COMPONENTS} REQUIRED)
```

Comment:
yes, if user happens to have 1.61 in his/her system, but he/she does not want to rebuild boost. then he/she will benefit from this.